README.md

```diff
@@ -37,40 +37,33 @@ The OpenSSH option `AuthorizedKeysCommand` cannot have any arguments
 specified following the command, and we need to pass options to
 specify which organization and team is used, as well as which user
 should be authenticated using keys in GitHub, so we will create a
-wrapper script for openssh-github-keys. Additionally
-`openssh-github-keys` will need your `GITHUB_TOKEN` which may be
-specified as a variable in the wrapper script, or in a
-[dotenv file](https://github.com/stackbuilders/dotenv-hs). Your
-wrapper script may look like the one below, which you could create as
-`/usr/local/bin/openssh-github-keys-wrapper`:
+configuration file for openssh-github-keys. Additionally
+`openssh-github-keys` will need your `GITHUB_TOKEN` which must be
+specified inside the `/etc/openssh-github-keys/github.creds` file. Your
+configuration file may look like the one below, which you create as
+`/etc/openssh-github-keys/login.conf`:
 
 ```bash
-#!/bin/bash
-
-/usr/local/bin/openssh-github-keys \
-    -o your-github-org \
-    -t your-github-team \
-    -u user-to-auth-with-github \
-    -f /path/to/your/github-token-dotenv-file \
-    $1
+organization your-github-org
+team your-github-team
+users user-to-auth-with-github ...
 ```
 
-Make sure this script is executable and owned by the `root` user (`chmod 700
-/usr/local/bin/openssh-github-keys-wrapper`).
+Make sure this file is owned by the `nobody` user (`chmod 0600
```

Comment:
I'd delete this line I think. Ownership of most files is root in /etc, and I think it's fine here as long as it's 644 or something. This file isn't sensitive, but the creds file is, and it should be owned by nobody and 600.